.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!-- Thank you for contributing to KusionStack!
+
+Note: 
+
+1. With pull requests:
+
+    - Open your pull request against "main"
+    - Your pull request should have no more than three commits, if not you should squash them.
+    - It should pass all tests in the available continuous integration systems such as GitHub Actions.
+    - You should add/modify tests to cover your proposed code changes.
+    - If your pull request contains a new feature, please document it on the README.
+
+2. Please create an issue first to describe the problem.
+
+    We recommend that link the issue with the PR in the following question.
+    For more info, check https://kusionstack.io/docs/governance/contribute/
+-->
+
+#### What type of PR is this?
+
+<!--
+Add one of the following kinds:
+/kind bug
```

Comment:
we don't have labels for these kinds...you need to rewrite it to be consistent to fact

Reply:
I have updated PR labels and kinds here